ARGO/include/Game.h

```diff
@@ -55,6 +58,12 @@ class Game
 	// bool for if game is running or not
 	bool m_isRunning;
 
+	std::vector<Controller*> controllers;
+
+	InputHandler m_inputHandler;
```

Comment:
Empty lines, line breaks aren't quite clear why things are broken up. Is there  a logic to the grouping?